src/js/components/Card.js

```diff
@@ -193,8 +193,15 @@ export default class Card extends Component {
     let link = this._renderLink();
     let videoLayer = this._renderVideoLayer();
 
+    const contentClasses = classnames(
+      {
+        [`${CLASS_ROOT}__content`]: true,
+        [`${CLASS_ROOT}__content--truncated`]: truncate
```

Comment:
Hi @ericsoderberghp, I've updated the PR to align the class label with the property name.